mypy.ini

```diff
@@ -1,2 +1,17 @@
 [mypy]
 strict = True
+
```

Comment:
I don't think any of these should be needed.  Having inline tags is much clearer and allows for removal when we reach that state much more accessible.